calico-policies/private-network-isolation/calico-v3/au-syd/deny-all-private-default.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: deny-all-private-default
+spec:
+  egress:
+  - action: Deny
+    destination: {}
+    source: {}
+  order: 99900
```

Comment:
I think we want to deny all ingress in this policy as well (like the policy we apply on the 1.16 update if the customer is already using private HEPs).  This way clusters that have been using private HEPs and policies in 1.15 and earlier, and those that don't add them until after they update/deploy 1.16, will have the same deny-all-private-default policy.